lib/commands/clipboard.js

```diff
@@ -0,0 +1,43 @@
+import { exec } from 'teen_process';
+
+const commands = {};
+
+/**
+ * Sets the clipboard's content on the Windows PC.
+ *
+ * @param {!string} content - The content to be set as base64 encoded string.
+ * @param {?string} contentType [plaintext] - The type of the content to set.
+ *                                            Only `plaintext` and 'image' are supported.
+ */
+commands.setClipboard = async function setClipboard (content, contentType) {
+  if (contentType === 'plaintext') {
+    await exec('powershell', ['-command', `$str=[System.Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String('${content}'));Set-Clipboard -Value $str`]);
+  } else if (contentType === 'image') {
+    await exec('powershell', ['-command', `$img=[Drawing.Bitmap]::FromStream([IO.MemoryStream][Convert]::FromBase64String('${content}'));[System.Windows.Forms.Clipboard]::SetImage($img);$img.Dispose();`]);
```

Comment:
These lines are too long

Reply:
Split all as it make them easier to read